tests/backup/backup_test.go

```diff
@@ -8177,3 +8195,47 @@ func removeIntItemFromSlice(itemList []int, item []int) []int {
 	}
 	return itemList
 }
+
+func getAllBackupLocations(ctx context.Context) (map[string]string, error) {
+	backupLocationMap := make(map[string]string, 0)
+	backupDriver := Inst().Backup
+	backupLocationEnumerateRequest := &api.BackupLocationEnumerateRequest{
+		OrgId: orgID,
+	}
+	response, err := backupDriver.EnumerateBackupLocation(ctx, backupLocationEnumerateRequest)
+	if err != nil {
+		return nil, nil
+	}
```

Comment:
Same here, I think we would need a message here about no backup locations found for the org or something, don't you think?

Reply:
Done